runtime/src/main/resources/aes_sec

```diff
@@ -0,0 +1 @@
+uBdUx82vPHkDKb284d7NkjFoNcKWBuka
```

Comment:
should this be on a separate file only? It will be easier to overwrite if the value is a simple property instead of a file.

Reply:
@swaroopar Hi Swaroopar,This is the secret key file. I discussed it with xieyichao. We think it is more secure and easier to update the secret key in the file.

Reply:
@WangLiNaruto is it possible to update the file without delivering a new version?

Reply:
We should not pack a secret file into the jar.